pkg/kubelet/util/format/pod_test.go

```diff
@@ -0,0 +1,150 @@
+		{"field_normal_case", fakeCreatePod("test-pod", metav1.NamespaceDefault, "551f5a43-9f2f-11e7-a589-fa163e148d75"), "test-pod_default(551f5a43-9f2f-11e7-a589-fa163e148d75)"},
+	}
+
+	for _, testCase := range testCases {
+		realPod := Pod(testCase.pod)
+		assert.Equalf(t, testCase.expectedValue, realPod, "Failed to test: %s", testCase.caseName)
+	}
+}
+
+func TestPodAndPodDesc(t *testing.T) {
+	testCases := []struct {
+		caseName      string
+		podName       string
+		podNamesapce  string
+		podUID        types.UID
+		expectedValue string
+	}{
+		{"field_empty_case", "", "", "", "_()"},
+		{"field_normal_case", "test-pod", metav1.NamespaceDefault, "551f5a43-9f2f-11e7-a589-fa163e148d75", "test-pod_default(551f5a43-9f2f-11e7-a589-fa163e148d75)"},
+	}
+
+	for _, testCase := range testCases {
+		realPodDesc := PodDesc(testCase.podName, testCase.podNamesapce, testCase.podUID)
+		assert.Equalf(t, testCase.expectedValue, realPodDesc, "Failed to test: %s", testCase.caseName)
+	}
+}
+
+func TestPodWithDeletionTimestamp(t *testing.T) {
+	normalDeletionTime := metav1.Date(2017, time.September, 26, 14, 37, 50, 00, time.UTC)
+
+	testCases := []struct {
+		caseName               string
+		isdeletionTimestampNil bool
+		deletionTimestamp      metav1.Time
+		expectedValue          string
+	}{
+		{"timestamp_is_nil_case", true, normalDeletionTime, "test-pod_default(551f5a43-9f2f-11e7-a589-fa163e148d75)"},
+		{"timestamp_is_normal_case", false, normalDeletionTime, "test-pod_default(551f5a43-9f2f-11e7-a589-fa163e148d75):DeletionTimestamp=2017-09-26T14:37:50Z"},
+	}
+
+	for _, testCase := range testCases {
+		fakePod := fakeCreatePodWithDeletionTimestamp("test-pod", metav1.NamespaceDefault, "551f5a43-9f2f-11e7-a589-fa163e148d75", &testCase.deletionTimestamp)
+
+		if testCase.isdeletionTimestampNil {
+			fakePod.SetDeletionTimestamp(nil)
+		}
+
+		realPodWithDeletionTimestamp := PodWithDeletionTimestamp(fakePod)
+		assert.Equalf(t, testCase.expectedValue, realPodWithDeletionTimestamp, "Failed to test: %s", testCase.caseName)
+	}
+}
+
+func TestPods(t *testing.T) {
+	pod1 := fakeCreatePod("pod1", metav1.NamespaceDefault, "551f5a43-9f2f-11e7-a589-fa163e148d75")
+	pod2 := fakeCreatePod("pod2", metav1.NamespaceDefault, "e84a99bf-d1f9-43c2-9fa5-044ac85f794b")
+
+	testCases := []struct {
+		caseName      string
+		pods          []*v1.Pod
+		expectedValue string
+	}{
+		{"input_nil_case", nil, ""},
+		{"input_empty_case", []*v1.Pod{}, ""},
+		{"input_length_one_case", []*v1.Pod{pod1}, "pod1_default(551f5a43-9f2f-11e7-a589-fa163e148d75)"},
+		{"input_length_more_than_one_case", []*v1.Pod{pod1, pod2}, "pod1_default(551f5a43-9f2f-11e7-a589-fa163e148d75), pod2_default(e84a99bf-d1f9-43c2-9fa5-044ac85f794b)"},
+	}
+
+	for _, testCase := range testCases {
+		realPods := Pods(testCase.pods)
+		assert.Equalf(t, testCase.expectedValue, realPods, "Failed to test: %s", testCase.caseName)
+	}
+}
+
+func TestPodsWithDeletiontimestamps(t *testing.T) {
```

Comment:
I prefer `TestPodsWithDeletionTimestamps`. (Also rename the original func to `PodsWithDeletionTimestamps`)

Reply:
It should be more standardized.
But, they are not the same contents, so i split them into different commits, thank you!